app/views/feeds/_form_apple_config.html.erb

```diff
@@ -27,9 +27,34 @@
           <%= key_fields.hidden_field :provider_id, data: {apple_key_target: "provider"} %>
           <%= key_fields.hidden_field :key_id, data: {apple_key_target: "key"} %>
         <% end %>
-      <% end %>
 
-      <%= form.hidden_field :type, value: "Feeds::AppleSubscription" %>
+        <div class="col-12 mt-4">
+          <div class="form-check">
+            <%= config_form.check_box :publish_enabled %>
+            <div class="d-flex align-items-center">
+              <%= config_form.label :publish_enabled %>
+              <%= help_text t(".help.publish_enabled") %>
+            </div>
+          </div>
+          <div class="form-check">
+            <%= config_form.check_box :sync_blocks_rss %>
+            <div class="d-flex align-items-center">
+              <%= config_form.label :sync_blocks_rss %>
+              <%= help_text t(".help.sync_blocks_rss") %>
+            </div>
+          </div>
+        </div>
+      <% end %>
+      <div class="col-12 mt-4">
+        <div class="form-floating">
+          <% if !@apple_show_options.blank? %>
+            <%= form.select :apple_show_id, @apple_show_options, {include_blank: true} %>
+          <% else %>
+            <%= form.text_field :apple_show_id %>
+          <% end %>
+          <%= form.label :apple_show_id, "Apple Show ID" %>
+        </div>
+      </div>
```

Comment:
Yeah, I thought about that, and I was torn on whether or not it was worth it - in the case where a user already knows the apple id, we don't need to make it two steps, and since almost all shows will have an apple id whether they have a subscription channel or not, I didn't want to prevent using a known value until it could be looked up.

I'll try making this conditional/two-step, and see how it goes!

Reply:
Another problem with this design is you can't delete or replace the API keys.